dietpi/dietpi-software

```diff
@@ -7066,10 +7080,13 @@ _EOF_
 
 	Apply_Logging_Choices(){
 
-		#Work out which Logging system needs installing from Indexs (if any)
-		#Work out which Logging system needs removing (if any), and, create a temp script file
+		#Work out which Logging system needs installing from IDs (if any)
+		#Work out which Logging system needs removing (if any)
 		if (( $INDEX_LOGGING_TARGET != $INDEX_LOGGING_CURRENT )); then
 
+			# Run uninstall of old logging systems, after install loop
+			uninstall_required=1
```

Comment:
I'd prefer this as a global in the script?
```
UNINSTALL_REQUIRED
```

We could then flag it as needed across the whole script?

Reply:
1 sec, mis read code. Checking again.

Reply:
@Fourdee 
Generally yes. In this case it is really just to handle the uninstall marks due to choice system. There is currently no other case for required uninstall between install and configure loops.